docs/index.md

```diff
@@ -9,42 +9,53 @@ has_toc: false
-Many of these detectable errors, like [buffer overflow](https://en.wikipedia.org/wiki/Buffer_overflow),
-can have serious security implications. Google has found [thousands] of security vulnerabilities and
-stability bugs by deploying
-[guided in-process fuzzing of Chrome components](https://security.googleblog.com/2016/08/guided-in-process-fuzzing-of-chrome.html),
-and we now want to share that service with the open source community.
-
+[Fuzz testing] is a well-known technique for uncovering programming errors in
+software. Many of these detectable errors, like [buffer overflow], can have
+serious security implications. Google has found [thousands] of security
+vulnerabilities and stability bugs by deploying [guided in-process fuzzing of
+Chrome components], and we now want to share that service with the open source
+community.
+
+[Fuzz testing]: https://en.wikipedia.org/wiki/Fuzz_testing
+[buffer overflow]: https://en.wikipedia.org/wiki/Buffer_overflow
 [thousands]: https://bugs.chromium.org/p/chromium/issues/list?q=label%3AStability-LibFuzzer%2CStability-AFL%20-status%3ADuplicate%2CWontFix&can=1
+[guided in-process fuzzing of Chrome components]: https://security.googleblog.com/2016/08/guided-in-process-fuzzing-of-chrome.html
+
+In cooperation with the [Core Infrastructure Initiative], OSS-Fuzz aims to make
+common open source software more secure and stable by combining modern fuzzing
+techniques with scalable, distributed execution.
+
+[Core Infrastructure Initiative]: https://www.coreinfrastructure.org/
+
+We support the [libFuzzer], [AFL], and [Honggfuzz] fuzzing engines in
+combination with [Sanitizers], as well as [ClusterFuzz], a distributed fuzzer
+execution environment and reporting tool.
 
-In cooperation with the [Core Infrastructure Initiative](https://www.coreinfrastructure.org/), 
-OSS-Fuzz aims to make common open source software more secure and stable by
-combining modern fuzzing techniques with scalable,
-distributed execution.
+[libFuzzer]: http://llvm.org/docs/LibFuzzer.html
+[AFL]: https://lcamtuf.coredump.cx/afl/
+[Honggfuzz]: https://github.com/google/honggfuzz
+[Sanitizers]: https://github.com/google/sanitizers
+[ClusterFuzz]: https://github.com/google/clusterfuzz
 
-We support the [libFuzzer](http://llvm.org/docs/LibFuzzer.html), [AFL](https://lcamtuf.coredump.cx/afl/) and
-[Honggfuzz](https://github.com/google/honggfuzz) fuzzing engines
-in combination with [Sanitizers](https://github.com/google/sanitizers), as well as
-[ClusterFuzz](https://github.com/google/clusterfuzz),
-a distributed fuzzer execution environment and reporting tool. 
+Currently, OSS-Fuzz supports C/C++, Rust, and Go code. Other languages supported
+by [LLVM] may work too. OSS-Fuzz supports fuzzing x86_64 and i386 builds.
 
-Currently, OSS-Fuzz supports C/C++, Rust, and Go code. Other languages supported by [LLVM](http://llvm.org) may work too.
-OSS-Fuzz supports fuzzing x86_64 and i386 builds.
+[LLVM]: http://llvm.org
```

Comment:
https

Reply:
resolved in 6015ee628a1034346128bde195a58cd139ea5d6b